data/processed/hebci/gbp_business_consulting.csv

```diff
@@ -0,0 +1,161 @@
+year,nuts_id,nuts_year_spec,gbp_business_consulting
+2014,UKH2,2018,5431000
```

Comment:
seems like these need autonuts

Reply:
Yup. I only applied autonuts to the merged indicators. Will get all these sorted now.